Allow reading message from stdin by passing -

paulgessinger opened the pull request:
I needed this feature for the CLI so I modified the executable to allow setting the `message` argument to "-". This overwrites `args.message` with stdin.

This allows me to do stuff like
```bash
./long-running-prog ; tail -n 20 prog.log | pushover --title "done" -
```

paulgessinger committed:
enable reading message from stdin by passing -

Thibauth commented:
Thanks for the PR. I would definitely like to allow reading input from STDIN. This might sound nitpicking, but your fix would break in the (unlikely) case where the user wants to send a message whose content is literally "-". I think better fixes could be (in my order of preference):

1. add a `--file` switch to read from a given filename; when the filename is `-`, then read from STDIN
1. keep the current behaviour, but when the input is empty, the command reads it from STDIN
1. add a `--stdin` switch to specify that the message should be read from STDIN

If you look at how other UNIX commands work, interpreting `-` as meaning "read from STDIN" is only done in contexts when a filename is expected (not a string). What do you think?

paulgessinger commented:
Somehow I missed your comment here, sorry!

You're probably right. I guess I agree with you that the `--file` argument probably is preferable. I'll update this PR.

paulgessinger committed:
adapt to PR comments

add additional argument —file, which is mutually exclusive with message. This can be used to load a message from a file, or stdin if - is passed.
